### PR TITLE
webgpu: support asin and acos operators

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6949)

### CODE REVIEW

**Review by qjia7 — COMMENTED** [review]
Why are these two `Acos.ts` and `Asin.st` files missing under src/kernels?

**Comment by xhcao** [comment]
Sorry, I forgot to add these two new files

File: tfjs-backend-webgpu/src/kernels/Acos.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC. All Rights Reserved.
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/Acos.ts:3]: nit: Remove `All Rights Reserved.` in the latest new added file. And similar for other places.

**Review by qjia7 — APPROVED** [review]
LGTM with a nit.

**Review by gyagp — APPROVED** [review]
LGTM

**Pull request merged**